docker/dockerfile-build-ubuntu-rock

```diff
@@ -35,8 +35,8 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-ins
 RUN pip3 install wheel && pip3 install tox pyyaml
 
 # Patch hcc-config.cmake in /opt/rocm
-ADD docker/scripts/fix_hcc_config /tmp
-RUN /tmp/fix_hcc_config && rm /tmp/fix_hcc_config
+#ADD docker/scripts/fix_hcc_config /tmp
+#RUN /tmp/fix_hcc_config && rm /tmp/fix_hcc_config
```

Comment:
Not deleting?

Reply:
Delete the script as well?

Reply:
I'm just leaving it there for now in case people are still building on 2.6